### DISCUSSION TITLE
怎么开启https?

### DISCUSSION THREAD

**Discussion by dxdigg** [AlistGo/alist#1041] [Q&A]
看了文档的scheme，但是不知道在修改https和上传key，用的是群晖安装的套件。

**Comment by xhofe** [comment]
套件不是由我制作的，我也不知道。问作者

**Reply by briheart** [reply]
能否在配置里增加一个开启https的选项，谢谢。folder sync pro这种好像只支持https webdav,不开不行。

**Comment by xiexiaofeng520** [comment]
我修改了https，证书也上传了，网页打不开

**Comment by cancundeyingzi** [comment]
我用的小皮面板，开启https了，正常访问

**Reply by 2250117806** [reply]
怎么配置的
